docs/api/desktop-capturer.md

```diff
@@ -100,6 +99,20 @@ Returns `Promise<DesktopCapturerSource[]>` - Resolves with an array of [`Desktop
 [`navigator.mediaDevices.getUserMedia`]: https://developer.mozilla.org/en/docs/Web/API/MediaDevices/getUserMedia
 [`systemPreferences.getMediaAccessStatus`]: system-preferences.md#systempreferencesgetmediaaccessstatusmediatype-windows-macos
 
+## Loopback Audio on macOS 13/14/15
```

Comment:
How about "13+"?

Reply:
I like that. I used "13+" in my revision.